Worker.Extension.Sql/src/SqlInputAttribute.cs

```diff
@@ -0,0 +1,56 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using Microsoft.Azure.Functions.Worker.Extensions.Abstractions;
+
+namespace Microsoft.Azure.Functions.Worker.Extension.Sql
+{
+    public sealed class SqlInputAttribute : InputBindingAttribute
+    {
+        /// <summary>
+        /// Constructs a new instance.
+        /// </summary>
+        /// <param name="commandText">The text of the command.</param>
+        /// <param name="commandType">Specifies whether commandText refers to a stored procedure or SQL query string.</param>
+        /// <param name="connectionStringSetting">The name of the app setting where the SQL connection string is stored.</param>
+        public SqlInputAttribute(string commandText, System.Data.CommandType commandType, string connectionStringSetting)
+        {
+            this.CommandText = commandText;
+            this.CommandType = commandType;
+            this.ConnectionStringSetting = connectionStringSetting;
+        }
+
+        /// <summary>
+        /// The name of the app setting where the SQL connection string is stored
+        /// (see https://docs.microsoft.com/en-us/dotnet/api/microsoft.data.sqlclient.sqlconnection?view=sqlclient-dotnet-core-2.0).
```

Comment:
Clean up any URLs you have in docs : 

- Remove lang IDs such as en-us
- Remove any unnecessary query parameters

Reply:
I just copy pasted from Sql Attribute, should we clean up there also?